extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/UndertowDeploymentRecorder.java

```diff
@@ -221,16 +220,13 @@ public static SocketAddress getHttpAddress() {
             Class<?> servletClass,
             boolean asyncSupported,
             int loadOnStartup,
-            BeanContainer beanContainer, Map<String, String> initParams,
+            BeanContainer beanContainer,
```

Comment:
I am not sure it's a good idea to change the API here. Other extensions (external to core - camel?) may be using it

Reply:
@gastaldi Hmm, such external extension would have to be fiddling with how Servlet is built in Quarkus *and* use Undertow extension for that...

If you have any suggestion what area such extension might appear in, I will try to find it and run its tests...

Reply:
Check if camel-quarkus is using it, otherwise it may be good to go then

Reply:
@gastaldi I run quarkus-camel tests, excluding Mongo (does not build with Quarkus master, talked to @ppalaga ) and JMS (local network shenanigans), and it looks unaffected, e.g. org.apache.camel.quarkus.component.servlet.CamelServletTest is :heavy_check_mark:  etc.

+ basic search for usage of said API did not show anything...

IMHO safe to go.